core/src/main/java/org/polypheny/db/functions/spatial/GeoDistanceFunctions.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2019-2024 The Polypheny Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.polypheny.db.functions.spatial;
+
+import org.jetbrains.annotations.NotNull;
+import org.locationtech.jts.geom.Coordinate;
+import org.locationtech.jts.operation.distance.DistanceOp;
+import org.polypheny.db.type.entity.spatial.PolyGeometry;
+
+
+/**
+ * Calculate the spherical distances between various geometries on the <strong>PERFECT SPHERE</strong>
+ */
+public class GeoDistanceFunctions {
+
+    // Define the radius of the Earth's sphere (in meters)
+    private static final double EARTH_RADIUS_KM = 6371.0 * 1000;
```

Comment:
The comment does not match the variable name, but its value.

Reply:
fixed